utils/utils.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+import requests
+
```

Comment:
Let's add
```
import time
```
here, so we can use the sleep method below.

Reply:
done